### DISCUSSION TITLE
Не работает расширение свободного пространства на Hyper-V (Windows Server 2019)

### DISCUSSION THREAD

**Discussion by Afistotel** [mikopbx/Core#449] [General ( Обо всём )]
При выборе "Увеличить размер диска хранения данных, задействовав всю неразмечанную область", просто пустой экран.

За 2 часа ничего не происходит. Расширял с 20 Гб до 40 Гб

**Comment by boffart** [comment]
Попробуйте вручную выполнить инструкции:
https://wiki.mikopbx.ru/faq:newdisk_vm

Мб станет ясно, в чем проблема. К примеру проверка диска на ошибки выполняется долго. 
Перед выполнением обязательно сделайте снэпшот машины.

**Comment by Afistotel** [comment]
# /etc/rc/freestorage 
Затык на этой команде. freestorage в папке нет
![image](https://user-images.githubusercontent.com/121282426/209378357-423b8857-be41-4d6d-b9e6-84bb885326f7.png)
![image](https://user-images.githubusercontent.com/121282426/209378553-cc644ad1-9fd2-4306-9510-de71c7a31ab3.png)

**Comment by boffart** [comment]
Да, устарела инструкция
Скрипт был перенесен. 
просто вбейте "freestorage" без указания полного пути.
